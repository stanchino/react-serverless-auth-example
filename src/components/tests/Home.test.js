import React from "react";
import { Home } from "..";
import { testComponent } from "./shared-examples";

describe("Home", () => testComponent(<Home />));