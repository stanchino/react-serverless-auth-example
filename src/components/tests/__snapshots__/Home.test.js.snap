// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without errors 1`] = `
<div>
  Home
</div>
`;
